apiary.apib

```diff
@@ -542,7 +542,186 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentor Request | Create a New Mentor Request [/api/v1/requests{?user_id,details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
++ Parameters
+    
+    + user_id (integer, required) - Identification of the User requesting a mentor in the form of an integer
```

Comment:
Since the passed `user` must also be verified, perhaps also include that information in this description.  i.e. "The passed in user must be a verified military veteran (i.e. `user.verified == true`)."